services/movie_session.py

```diff
@@ -0,0 +1,45 @@
+from db.models import MovieSession
+from datetime import datetime
+
+
+def create_movie_session(
+    movie_show_time: datetime, movie_id: int, cinema_hall_id: int
+) -> MovieSession:
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+    return movie_session
+
+
+def get_movies_sessions(session_date: datetime = None) -> list[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+    session_id: int,
+    show_time: datetime = None,
+    movie_id: int = None,
+    cinema_hall_id: int = None,
+) -> MovieSession:
+    movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
Using `MovieSession.objects.get(id=session_id)` will raise a `DoesNotExist` exception if no session with the given ID is found. Consider handling the exception to avoid unhandled exceptions, especially if this function is called from a context where such exceptions are not expected.